src/System.Private.ServiceModel/src/windows/project.json

```diff
@@ -57,7 +57,8 @@
     },
     "netcore50": {
       "dependencies": {
-        "System.Runtime.WindowsRuntime": "4.0.11-rc2-23608"
+        "System.Runtime.WindowsRuntime": "4.0.11-rc2-23608",
+        "Microsoft.TargetingPack.Private.NetNative": "1.0.0-rc2-23607"
```

Comment:
The reason I added this target here is because this project needed to reference Windows.winmd, which is included in our "NetNative" targeting pack. This project uses contract references "System.Runtime.WindowsRuntime" and therefore doesn't need any of the targeting pack references for CoreCLR (i.e. mscorlib.dll)